Essentials/src/messages_en.properties

```diff
@@ -497,6 +497,7 @@ teleportationEnabled=\u00a76Teleportation \u00a7cenabled\u00a76.
 teleportationEnabledFor=\u00a76Teleportation \u00a7cenabled \u00a76for \u00a7c{0}\u00a76.
 teleportAtoB=\u00a7c{0}\u00a76 teleported you to \u00a7c{1}\u00a76.
 teleportDisabled=\u00a7c{0} \u00a74has teleportation disabled.
+teleportDown=\u00a76Teleporting down.
```

Comment:
This localization needs copied to the other localization language files

Reply:
This isn't the case - Crowdin will do this.